client/api/system/v1alpha1/api.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package v1alpha1;
+
+option go_package = "github.com/kubernetes-csi/csi-proxy/client/api/system/v1alpha1";
+
+service System {
+    // GetSerialNumber returns the device's serial number
```

Comment:
Can you rename it to `GetBIOSSerialNumber` please? I think there will be other system-wide serial numbers we will want to query down the line and having BIOS in the name makes it more specific.